modules/tables/src/lib/table/row-table-batch.js

```diff
@@ -16,12 +21,16 @@ export default class RowTableBatch {
     }
   }
 
-  addRow(row) {
+  addRow(row, cursor) {
     if (!this.rows) {
       this.rows = new Array(this.batchSize);
       this.length = 0;
     }
-    this.rows[this.length] = convertRowToObject(row, this._headers);
+    if (Number.isFinite(cursor)) {
+      this.cursor = cursor;
+    }
+
+    this.rows[this.length] = this.convertToObject ? convertRowToObject(row, this._headers) : row;
```

Comment:
Ideally the batch handler should not know much about the content of the batches - it seems to me that the CSV parser should be responsible for the convertToObject logic. 

This is based on the assumption that most other table parsers will not have this issue.

Reply:
But CSVLoader sends rows to this `RowTableBatch` and `convertRowToObject` is called inside `RowTableBatch` by default. there is no reason `RowTableBatch` always converts rows to array of objects. especially CSVLoader allows `header: false` as  a csv prop, and it currently doesn't work, because of this conversion here

Reply:
Oh you mean move this `convertToObject` to CSV parser, so no conversion will happen in `RowTableBatch`?